offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/create/CreateProductView.groovy

```diff
@@ -113,29 +116,29 @@ class CreateProductView extends HorizontalLayout{
 
     private void bindViewModel(){
         //bind all textfields
-        this.productNameField.addValueChangeListener({this.createProductViewModel.productName = it.value })
+        this.productNameField.addValueChangeListener({this.viewModel.productName = it.value })
```

Comment:
Imagine a world where this would say createProductViewModel. I'd like to live in that world 😄 
It's completely fine to just call it viewModel though

Reply:
👀